app/src/main/java/com/github/sdp/mediato/utility/PhotoPicker.java

```diff
@@ -0,0 +1,65 @@
+package com.github.sdp.mediato.utility;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import androidx.activity.result.ActivityResult;
+import androidx.activity.result.ActivityResultCallback;
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.ActivityResultRegistry;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import com.github.dhaval2404.imagepicker.ImagePicker;
+
+public class PhotoPicker {
+
+  private final Fragment fragment;
+  private final ImageView imageView;
+  private Uri profileImageUri;
+
+  public PhotoPicker(@NonNull Fragment fragment, @NonNull ImageView imageView) {
+    this.fragment = fragment;
+    this.imageView = imageView;
+  }
+
+  public View.OnClickListener getOnClickListener(ActivityResultRegistry activityResultRegistry) {
+    ActivityResultLauncher<Intent> photoPickerResult = registerForPhotoPickerResult(activityResultRegistry);
+    return v -> ImagePicker.Companion.with(fragment)
+        .crop()
+        .cropSquare()
+        .compress(1024)
+        .maxResultSize(620, 620)

```

Comment:
Is not 620x620 pixels a little too big ? Particularly if we display them only with a static 156x156 pixels size